pyproject.toml

```diff
@@ -212,6 +211,10 @@ line-length = 88
 # Don't automatically concatenate strings -- sometimes we forget a comma!
 unfixable = ["ISC"]
 
+[tool.ruff.format]
+quote-style = "double"
+exclude = ["migrations/versions/*"]
```

Comment:
YES thank you for excluding the generated files! This always mildly bugged me and then I would forget to fix it 🙄